jdaviz/configs/imviz/plugins/conesearch/conesearch.py

```diff
@@ -0,0 +1,30 @@
+from astropy.table import Table
+from astropy.coordinates import SkyCoord
+from astroquery.sdss import SDSS
+
+from jdaviz.core.registries import tray_registry
+from jdaviz.core.template_mixin import PluginTemplateMixin, ViewerSelectMixin
+
+__all__ = ['ConeSearch']
+
+
+@tray_registry('imviz-conesearch', label="Imviz ConeSearch")
+class ConeSearch(PluginTemplateMixin, ViewerSelectMixin):
+    template_file = __file__, "conesearch.vue"
+
+    def __init__(self, *args, **kwargs):
+        self.init_astrowidgets_api()
+        super().__init__(*args, **kwargs)
+
+    def vue_do_conesearch(self, *args, **kwargs):
+        query_region_result = SDSS.query_region(SkyCoord('00h07m13.8036s +14d56m16.8706s', frame='icrs'), radius='0.015 deg', data_release=17)
+
+        skycoord_table = SkyCoord(query_region_result['ra'], query_region_result['dec'], unit='deg')
+
+        conesearch_results = Table({'coord': [skycoord_table]})
+        self.add_markers(table=conesearch_results, use_skycoord=True, marker_name='conesearch_results')
```

Comment:
Think about the use case where someone click "Conesearch" button a few times in a row.

* Do we want to clear the old results or append to them?
* What if it is the exact same query? Do we want to cache what we have and not repeat the same query over and over again?
* How do we prevent DoS attack when someone just keeps pushing that button? (Maybe Jesse's baby got ahold of the plugin...)

Reply:
A more complex use case:

* User zoom into an area, user queries the catalog, we have the results.
* User pan just a little, user queries the catalog again. Now the results will have overlap but also new stuff. How do we consolidate this? (Depends on whether we want to overwrite or append.)